include/aws/common/error.h

```diff
@@ -14,6 +14,7 @@
 
 #define AWS_OP_SUCCESS (0)
 #define AWS_OP_ERR (-1)
+#define AWS_OP_SKIP (1)
```

Comment:
probably not the best place to define a test only return code. can we put it in some test related header?

Reply:
I can move the AWS_OP_SKIP into aws_test_harness.h. However, `AWS_OP_SUCCESS` and `AWS_OP_ERR ` has been used in all the current tests, it makes more sense to have them together. 
Ideally, we should define test specified codes for succes/err/skip in aws_test_harness.h. However, that will be a significant API change that affects all aws-c tests, I dont know if we want to do it now?

Reply:
AWS_OP_SUCCESS and AWS_OP_ERR are also used extensively outside of tests. My concern is we are adding a test specific constant into a common header. In my opinion we should not be polluting this header with test only stuff and putting just the skip macro in test harness is ok as compromise, but we can have a team wide discussion on this.